### PR TITLE
Fix #3041

### LINKED ISSUE #3041: unknown API-MS-Win pseudo-dll
Hi,
I got this error:

```
[..]
<unknown API-MS-Win pseudo-dll api-ms-win-core-localization-obsolete-l1-2-0.dll>
<unknown API-MS-Win pseudo-dll api-ms-win-core-kernel32-private-l1-1-0.dll>
<Application C:\crackme_xor.exe (15740). Unable to load client library: import RegisterWaitForInputIdle not found in KERNELBASE.dll.>
```
I've checked and `RegisterWaitForInputIdle` its at `kernel32.dll`:
```
>dumpbin /exports "C:\Windows\System32\kernel32.dll" | grep RegisterWaitForInputIdle
       1190  4A5 00010B10 RegisterWaitForInputIdle
```

Should I do a PR adding both DLLs (`pseudo-dll api-ms-win-core-localization-obsolete-l1-2-0.dll` and `pseudo-dll api-ms-win-core-kernel32-private-l1-1-0.dll`) in here 

https://github.com/illera88/dynamorio/blob/bb3dc9bd4dd516ce849ad80a97a6108ca0606fc0/core/win32/loader.c#L1534
??

Regards

### PR DESCRIPTION
**Pull request by illera88**

### CODE REVIEW

**Comment by illera88** [comment]
New error arises:

```
<unknown API-MS-Win pseudo-dll api-ms-win-core-appinit-l1-1-0.dll>
<unknown API-MS-Win pseudo-dll api-ms-win-core-com-l1-1-0.dll>
<Application C:\Users\alberto.garcia\Documents\code\Athena\Athena\build_x64\bin\crackme_xor.exe (3540). Unable to load client library: import CoInitializeEx not found in KERNELBASE.dll.>
<unknown API-MS-Win pseudo-dll api-ms-win-core-com-l1-1-0.dll>
```
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional>dumpbin /exports "C:\Windows\System32\ole32.dll" | grep CoInitializeEx
        932   61          CoInitializeEx (forwarded to api-ms-win-core-com-l1-1-0.CoInitializeEx)
```
So need to add those two (`api-ms-win-core-appinit-l1-1-0.dll` and `api-ms-win-core-com-l1-1-0.dll`) to the loader.

Am I doing it right adding both of them @derekbruening ?

Cheers

File: core/win32/loader.c [hunk 1]
```diff
@@ -1532,10 +1532,15 @@ map_api_set_dll(const char *name, privmod_t *dependent)
              str_case_prefix(name, "API-MS-Win-Eventing-Provider-L1-1"))
         return "kernelbase.dll";
     else if (str_case_prefix(name, "API-MS-Win-Core-PrivateProfile-L1-1") ||
-             str_case_prefix(name, "API-MS-Win-Core-Atoms-L1-1"))
+             str_case_prefix(name, "API-MS-Win-Core-Atoms-L1-1") ||
+             str_case_prefix(name, "API-MS-Win-Core-Localization-Obsolete-L1-2") ||
+             str_case_prefix(name, "API-MS-Win-Core-Kernel32-private-L1-1"))
```
Comment (derekbruening) [inline core/win32/loader.c]: nit: to match the others: s/private/Private/

Comment (illera88) [reply core/win32/loader.c]: can you make that change and merge please?

thanks

**Comment by derekbruening** [comment]
Looks like these have all since been added by other patches.  API-MS-Win-Core-COM-L1-1 was added as combase.dll instead of the ole here, and API-MS-Win-Core-AppInit-L1-1 is also different, but it seems best to go with what's already checked in.

**Pull request closed without merging**